notebook/services/contents/fileio.py

```diff
@@ -30,6 +30,16 @@
     from base64 import encodestring as encodebytes, decodestring as decodebytes
 
 
+def replace_file(src, dst):
+    """ replace dst with src
+
+    switches between os.replace or os.rename based on python 2.7 or python 3
```

Comment:
This isn't super helpful ;-). When writing docstrings or comments, think about why someone might be reading it. Mentioning Python 3 and 2.7 is a good start, it suggests that one is better but not available on the older version, but it doesn't say which, nor why it's better.